.github/workflows/tests.yml

```diff
@@ -23,9 +23,9 @@ jobs:
         uses: actions/checkout@v2
       - uses: actions/setup-node@v2
         with:
-          node-version: '14.x'
+          node-version: '18.15'
```

Comment:
you have to update the jenkinsfile as well

Reply:
might as well use 18.16.1, to align with Jenkinsfile

Reply:
... unless 18.15 is the version supported by vscode

Reply:
18.15 is the version supported by vscode-extension-tester. It's not available (yet) on our Jenkins instance, but I'm going to add it. Luckily, our Jenkins script doesn't seem to run the UI tests (likely because we don't want the release to fail due to a flaky UI test), so using a newer version shouldn't be a problem for the time being.